hardhat.config.ts

```diff
@@ -52,6 +52,11 @@ const config: HardhatUserConfig = {
       accounts,
       chainId: 8453,
     },
+    {
```

Comment:
Can we also add an entry under `customChains` for etherscan/contract verification?